aspnetcore/mvc/views/razor.md

```diff
@@ -734,36 +734,59 @@ When set to `false` (default), whitespace in the rendered markup from Razor comp
 
 *This scenario only applies to Razor components (`.razor`).*
 
-<!-- UPDATE 8.0 RC2: Remove preview remark and update code -->
-
 Sets the render mode of a Razor component:
 
-* `Server`: Applies interactive server rendering using Blazor Server.
-* `WebAssembly`: Applies interactive client rendering using Blazor WebAssembly.
-* `Auto`: Initially applies interactive client rendering using Blazor Server, and then applies interactive client rendering using WebAssembly on subsequent visits after the Blazor bundle is downloaded.
+* `InteractiveServer`: Applies interactive server rendering using Blazor Server.
+* `InteractiveWebAssembly`: Applies interactive WebAssembly rendering using Blazor WebAssembly.
+* `InteractiveAuto`: Initially applies interactive WebAssembly rendering using Blazor Server, and then applies interactive WebAssembly rendering using WebAssembly on subsequent visits after the Blazor bundle is downloaded.
 
-The following example sets the Server render mode to a `Dialog` component:
+For a component instance:
```

Comment:
Technically `@rendermode` is both a Razor directive and a Razor directive attribute. The semantics are similar, but there are differences. The `@rendermode` directive is on the component definition, so the referenced render mode instance has to be static. The `@rendermode` directive attribute can take any render mode instance.